crawl-ref/source/dat/database/miscname.txt

```diff
@@ -136,6 +136,14 @@ The Orb of Zot is coveted for many reasons. Will you seize it for your own?
 A smirking vizier told you that the Orb of Zot is just within this dungeon.
 
 If you return without finding the Orb of Zot, so far below, it's all over!
+
+One, two, three... Ready or not, find the Orb of Zot!
+
+Far below lies the Orb of Zot. Hopefully nothing unpleasant happens on the way there.
```

Comment:
I like the concept of this one! The tone is a little bit too casual, though, and the opener is almost exactly the same as one above ("Below lies the Orb of Zot..."). I've tweaked it a bit.